src/Lecture2.hs

```diff
@@ -185,7 +233,9 @@ False
 True
 -}
 isIncreasing :: [Int] -> Bool
-isIncreasing = error "TODO"
+isIncreasing [] = True
+isIncreasing (_:[]) = True
+isIncreasing (x:y:rest) = if y > x then isIncreasing (y:rest) else False
```

Comment:
You can implement `isIncreasing` in a slightly more elegant way by relying on laziness in Haskell 🦥 

```suggestion
isIncreasing (x : y : rest) = x < y && isIncreasing (y : rest)
```